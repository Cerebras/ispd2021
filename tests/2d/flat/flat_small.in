# Dimension
2
# Volume X, Y
4 4
# Fabric
4 4
# Cost params
1 0 
# Heatmap
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1