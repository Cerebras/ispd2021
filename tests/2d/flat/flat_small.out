# Inverse sampling step
10
# Prisms
0 0 0 1 4
0 10 0 1 4
0 20 0 1 4
0 30 0 1 4
compute_map:
0 0
0 1
0 2
0 3
1 0
1 1
1 2
1 3
2 0
2 1
2 2
2 3
3 0 
3 1
3 2
3 3
adapter_map:
